web/js-file.js

```diff
@@ -1,6 +1,8 @@
 function funk(a,b) {
```

Comment:
Reporter: JSHINT
Rule: ?
Severity: WARN
File: ../../../web/js-file.js L1
 

&apos;funk&apos; is defined but never used.: function funk(a,b) {

<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>